lib/+wbs/getModelInfoFromRobotCfgAndYarpFinder.m

```diff
@@ -0,0 +1,10 @@
+function [modelPath,meshFilePrefix] = getModelInfoFromRobotCfgAndYarpFinder(robot_config)
+%getModelInfoFromRobotCfgAndYarpFinder Returns the model path and mex path prefix
+%   This function uses the Yarp Resource Finder for finding the model file.
+
+[~,modelPathFromYarpFinder] = system(['yarp resource --find ',robot_config.fileName]);
+modelPath = strcat(fileparts(char(eval(modelPathFromYarpFinder))),'/');
+meshFilePrefixSplit = strsplit(modelPath,'share');
+meshFilePrefix = strcat(strjoin(meshFilePrefixSplit(1:end-1),'share'),'share');
```

Comment:
I may be wrong, but `meshFilePrefix` should not be necessary anymore since iDynTree 3, see https://github.com/robotology/idyntree/releases/tag/v3.0.0 and https://github.com/robotology/idyntree/pull/817 .

Reply:
Indeed. Still, I have to set the parameter to some value, since it's a required input parameter of `+iDynTreeWrappers/prepareVisualization`. So I'll set to n empty char.